Provis.Core/Interfaces/Services/ICommentService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Provis.Core.DTO.CommentDTO;
+using Provis.Core.DTO.CommentsDTO;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Provis.Core.Interfaces.Services
+{
+    public interface ICommentService
+    {
+        Task<List<CommentListDTO>> GetCommentsAsync(int taskId);
+        Task CommentAsync(CreateCommentDTO commentDTO, string userId);
```

Comment:
I think that "CreateCommentAsync" or "AddCommentAsync" is more informative than "CommentAsync"